sdks/python/run_postcommit.sh

```diff
@@ -40,11 +39,15 @@ python setup.py sdist
 
 SDK_LOCATION=$(find dist/apache-beam-*.tar.gz)
 
+# Install test dependencies for ValidatesRunner tests.
+echo "pyhamcrest" > postcommit_requirements.txt
+echo "mock" >> postcommit_requirements.txt
+
 # Run integration tests on the Google Cloud Dataflow service
 # and validate that jobs finish successfully.
 echo ">>> RUNNING TEST DATAFLOW RUNNER it tests"
 python setup.py nosetests \
-  --attr IT \
+  --attr $1 \
```

Comment:
Could you verify that this command line option is passed (with a usage blurb if it isn't)?

Reply:
Done. Also added the ability to receive GCP project / path parameters to run tests locally.